src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/TTY.java

```diff
@@ -99,6 +104,14 @@ public void vmStartEvent(VMStartEvent se)  {
         MessageOutput.lnprint("VM Started:");
     }
 
+/*[IF CRIU_SUPPORT]*/
+    @Override
+    public void vmRestoreEvent(VMRestoreEvent se)  {
+        Thread.yield();  // fetch output
+        System.out.print("VM restored from checkpoint: ");
```

Comment:
Perhaps this should use `MessageOutput` like some other events:
```java
        MessageOutput.lnprint("VM restored from checkpoint:");
```

Reply:
Switched to using `MessageOutput.lnprint`, though that did require me to change another file `TTYResources.java`. Is that fine?

Reply:
I think that means you need to change `TTYResources_*.java` as well.

Reply:
Tested it and it pulls from the base resource bundle if it can't find the key in the specific locale bundle.

```
./openj9-openjdk-jdk/build/linux-x86_64-server-release/images/jdk/bin/jdb -attach localhost:8800
nicht abgefangene java.lang.Throwable festlegen
nicht abgefangene java.lang.Throwable mit Verzögerung festlegen
jdb wird initialisiert...
> 
VM restored from checkpoint: "Thread=main", org.eclipse.openj9.criu.CRIUSupport.checkpointJVMImpl(), Zeile=-1 BCI=-1
```